llvm/lib/Target/RISCV/RISCVInstrInfoCOREV.td

```diff
@@ -134,41 +148,74 @@ let Predicates = [HasExtXCoreVMac], hasSideEffects = 0, mayLoad = 0, mayStore =
 } // Predicates = [HasExtXCoreVMac], hasSideEffects = 0, mayLoad = 0, mayStore = 0, Constraints = "$rd = $rd_wb"
 
 
-let hasSideEffects = 0, mayLoad = 0, mayStore = 0 in
-class CVSIMDALUrr<bits<5> funct5, bit F, bit funct1, bits<3> funct3, string opcodestr>
+class CVSIMDALURR<bits<5> funct5, bit F, bit funct1, bits<3> funct3, string opcodestr>
     : CVInstSIMDRR<funct5, F, funct1, funct3, OPC_CUSTOM3, (outs GPR:$rd),
-              (ins GPR:$rs1, GPR:$rs2), opcodestr, "$rd, $rs1, $rs2"> {
+              (ins GPR:$rs1, GPR:$rs2), opcodestr, "$rd, $rs1, $rs2">;
+
+class CVSIMDALURRWb<bits<5> funct5, bit F, bit funct1, bits<3> funct3, string opcodestr>
+    : CVInstSIMDRR<funct5, F, funct1, funct3, OPC_CUSTOM3, (outs GPR:$rd_wb),
+              (ins GPR:$rd, GPR:$rs1, GPR:$rs2), opcodestr, "$rd, $rs1, $rs2"> {
+    string Constraints = "$rd = $rd_wb";
```

Comment:
They should be the same, `let` is a short cut for defining class members in multiple classes.

Reply:
Also it should have been a typo in #24, since this `let` only applied to `CVSIMDALUrr`.

https://github.com/openhwgroup/corev-llvm-project/blame/b18cb93b416dedbfe6e059f3ced2993334fbafda/llvm/lib/Target/RISCV/RISCVInstrInfoCOREV.td#L137

Reply:
Comment from @simonpcook that `let` is correct, since you are redefining an existing field.

Reply:
Fixed.